VK/VK/FriendListScreen/FriendListTableViewController.swift

```diff
@@ -53,114 +46,90 @@ final class FriendListTableViewController: UITableViewController, UIViewControll
 
     override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         guard
             let cell = tableView.dequeueReusableCell(
                 withIdentifier: CellIdentifiers.commonGroupTableViewCellID,
                 for: indexPath
-            ) as? CommonGroupTableViewCell else { return UITableViewCell() }
+            ) as? CommonGroupTableViewCell
+        else { return UITableViewCell() }
 
-        guard usersAtSections.count > 1 else { return UITableViewCell() }
-        cell.configureCell(unit: usersAtSections[indexPath.section][indexPath.row], labelNameTapHandler: tapHandler)
+        let sortedKeys = sortedFriendsMap.keys.sorted()
+        let key = sortedKeys[indexPath.section]
+        guard
+            let friendsListSection = sortedFriendsMap[key],
+            let friendsInfo = friendsListSection[safe: indexPath.row]
+        else { return UITableViewCell() }
+        cell.configureCell(unit: friendsInfo, labelNameTapHandler: tapHandler)
         return cell
     }
 
     // MARK: - Private methods
 
-    private func clearAllArrays() {
-        users.removeAll()
-        sortedUsers.removeAll()
-        lettersCountMap.removeAll()
-        sortedlettersMap.removeAll()
-        usersAtSections = [[]]
-        lettersMap = Alphabets.russianLettersMap
-    }
-
-    private func configScreen() {
-        configDotsLoader()
-        regCells()
-        configreTapHandler()
-    }
-
-    private func setupCell() {
-        makeUsers()
-        getFirstLettersCount()
-        calculateNumberOfSections()
-        sortLettersCountMap()
-        putContactsInSections()
-    }
-
-    private func makeUsers() {
-        var indexCounter = 0
-        for user in UserNames.names {
-            users.append(NetworkUnit(
-                name: user,
-                description: UserLocations.locations[safe: indexCounter] ?? "",
-                avatarImageName: UserAvatarImageNames.avatarImageNames[safe: indexCounter] ?? "",
-                unitImageNames: UserAvatarImageNames.avatarImageNames
-            ))
-            indexCounter += 1
-        }
+    private func configureScreen() {
+        fetchFriends()
+        configScreen()
     }
 
-    private func getFirstLettersCount() {
-        sortedUsers = users.sorted(by: { user1, user2 in
-            guard
-                let user1FirstLetter = user1.name.first,
-                let user2FirstLetter = user2.name.first
-            else { return false }
-            return user1FirstLetter < user2FirstLetter
-        })
-
-        for user in sortedUsers {
-            for letter in lettersMap where letter.key == user.name.first {
-                sortedlettersMap[letter.key] = letter.value + 1
-                lettersMap[letter.key]? += 1
+    private func fetchFriends() {
+        NetworkServiceble.shared.fetchFriends(
+            parametersMap: [
+                Constants.fieldsName: Constants.fieldsValue,
+            ]
```

Comment:
```suggestion
            parametersMap: [
            ]
```
VC не должен ничего знать о сетевых запросах!!!
параметры сетевых запросов внутри NetworkLayer (NetworkService)
вынести в сервис